File: stratstuff/python/src/IPCClient.py
import socket

class IPCClient():
    
    def __init__(self):
        self.HOST, self.PORT = "localhost", 32000
        self.sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
    
    def send(self, data):
        self.sock.sendto(data + "\n" , (self.HOST, self.PORT))

File: stratstuff/src/stratstuff/IPCClient.java
package stratstuff;

import java.io.IOException;
import java.net.DatagramPacket;
import java.net.DatagramSocket;
import java.net.InetAddress;
import java.net.SocketException;

public class IPCClient {

	DatagramSocket socket;

	public IPCClient() {
		try {
			socket = new DatagramSocket();
		} catch (SocketException e) {
			e.printStackTrace();
		}
	}

	public void send(String s, int port) {
		try {
			byte[] raw = s.getBytes();

			DatagramPacket packet = new DatagramPacket(raw, raw.length,
					InetAddress.getByName("localhost"), port);

			socket.send(packet);

		} catch (IOException e) {
			e.printStackTrace();
		}
	}
}
